cmd/helm3.go

```diff
@@ -308,8 +325,12 @@ func (d *diffCmd) template(isUpgrade bool) ([]byte, error) {
 	return filter(out), err
 }
 
-func (d *diffCmd) writeExistingValues(f *os.File) error {
-	cmd := exec.Command(os.Getenv("HELM_BIN"), "get", "values", d.release, "--all", "--output", "yaml")
+func (d *diffCmd) writeExistingValues(f *os.File, all bool) error {
```

Comment:
add some unittests for this func. please.

Reply:
I don't see a way to meaningfully test this function. I mean, it's basically a wrapper for `helm get values` which fetches stuff over the network. I had two ideas originally: either monkeypatch the OS calls, or refactor the function to use a callback that we can replace while testing. In both approaches we would be eliminating the fetching, which would make the test useless in effect.

I'm open to any alternative ideas…